larray/core.py

```diff
@@ -3676,7 +3688,7 @@ def percentile(array, *args, **kwargs):
      a0 |  0 |  1 |  2
      a1 |  3 |  4 |  5
     >>> percentile(arr, 25)
-    array(1.25)
+    1.25
```

Comment:
here percentile returns a ndarray with one element. 
I guess it's due to how _aggregate is implemented. 
I can replace by print(percentile(arr, 25)) to get a scalar.